### PR TITLE
Update vscode-languageserver and vscode-languageclient dependencies

### PR DESCRIPTION
**Pull request by debovema**
Fixes https://github.com/eclipse/sprotty-vscode/issues/48

### CODE REVIEW

File: sprotty-vscode-extension/package.json [hunk 1]
```diff
@@ -42,10 +42,11 @@
   "main": "lib/index",
   "types": "lib/index",
   "dependencies": {
+    "lerna": "^3.20.2",
```
Comment (spoenemann) [inline sprotty-vscode-extension/package.json:45]: `lerna` is a build tool and should not be in `dependencies`.

**Review by spoenemann — COMMENTED** [review]
Sorry for the late review! This repository was not on my radar.

Thanks for the valuable contribution. I have only one remark:

**Comment by spoenemann** [comment]
@debovema how come you are submitting a commit made by @MarianPalkus? I would prefer the commit authors to submit themselves.

Furthermore, the authors need to sign the [Eclipse Contributor Agreement](https://accounts.eclipse.org/user/eca) and sign-off their commits (`-s` flag) so they can be accepted to an Eclipse Foundation project.

How shall we proceed? @MarianPalkus would you ensure you can comply with the requirements above and create a new PR?

File: sprotty-vscode-extension/package.json (hunk 1 above)
Comment (MarianPalkus) [reply sprotty-vscode-extension/package.json:45]: ✅

**Comment by MarianPalkus** [comment]
Thanks for your review, @spoenemann. I will create a new PR.

**Comment by MarianPalkus** [comment]
I created a follow-up PR: #52

**Comment by debovema** [comment]
> @debovema how come you are submitting a commit made by @MarianPalkus? I would prefer the commit authors to submit themselves.

Sorry I submitted the PR because GitHub let me do it and to allow a review on this repo rather than losing the work being made on @MarianPalkus repository.

**Comment by spoenemann** [comment]
Ok no problem, I was just wondering.

**Pull request closed without merging**